api/models/helpers.go

```diff
@@ -389,3 +389,12 @@ func UpperName(name string) string {
 
 	return us
 }
+
+func StringInSlice(str string, slice []string) bool {
```

Comment:
I used them during initial design phase and thought they were nifty to have around. But think "less code is more" wins here.